Modules/Microsoft365DSC/DSCResources/MSFT_TeamsUpdateManagementPolicy/MSFT_TeamsUpdateManagementPolicy.schema.mof

```diff
@@ -0,0 +1,14 @@
+[ClassVersion("1.0.0.0"), FriendlyName("TeamsUpdateManagementPolicy")]
+class MSFT_TeamsUpdateManagementPolicy : OMI_BaseResource
+{
+    [Key, Description("Identity of the Teams Update Management Policy.")] String Identity;
+    [Write, Description("The description of the Teams Update Management Policy.")] String Description;
+    [Write, Description("Determines if managed updates should be allowed or not.")] Boolean AllowManagedUpdates;
+    [Write, Description("Determines if preview builds should be allowed or not.")] Boolean AllowPreview;
+    [Write, Description("Determines the ring of public previews to subscribes to."), ValueMap{"Disabled","Enabled","FollowOfficePreview"}, Values{"Disabled","Enabled","FollowOfficePreview"}] String AllowPublicPreview;
+    [Write, Description("Determines the day of week to perform the updates. Value shoud be between 0 and 6.")] UInt32 UpdateDayOfWeek;
+    [Write, Description("Determines the time of day to perform the updates. Must be a valid HH:MM format string with leading 0. For instance 08:30.")] String UpdateTime;
```

Comment:
Since the documentation for the cmdlet 'New-CsTeamsUpdateManagementPolicy' is missing: What is the difference between the UpdateTime and UpdateTimeOfDay parameters? The current descriptions don't show this.

Can you use both parameters at the same time? If not, shouldn't we add a check for that?

And if a leading 0 is required, shouldn't we check for that? Or add a leading 0 if it isn't provided?

Reply:
This part is not clear. My understanding is that Update time will overwrite UpdateTimeOfDay. I opted to include both for a reporting perspective. The Date part of UpdateTimeOfDay will always be set to today.